### DISCUSSION TITLE
Warning notifications when executing tests in the whole file

### DISCUSSION THREAD

**Discussion by zivkovicn** [fredrikaverpil/neotest-golang#185] [General questions]
Hello all, I'm just trying to determinate what is happening when I run all the tests, here is the screenshot that I captured:

![Screenshot 2024-09-11 at 5 10 38 PM](https://github.com/user-attachments/assets/bbfe0035-5f2b-48ab-b4d7-6be6ce49ef64)

The tests are running successfully. Just those warnings sometimes are all over the place.

I'm using LazyVim, and this is my configuratino:

```
local golangConfig = {
  go_test_args = {
    "-v",
    "-race",
    "-count=1",
    "-tags=test",
  },
}

return {
  { "nvim-neotest/neotest-plenary" },
  {
    "nvim-neotest/neotest",
    dependencies = {
      "fredrikaverpil/neotest-golang",
      "nvim-neotest/neotest-plenary",
    },
    opts = {
      discovery = {
        -- Drastically improve performance in ginormous projects by
        -- only AST-parsing the currently opened buffer.
        enabled = false,
      },
      adapters = {
        "neotest-plenary",
        require("neotest-golang")(golangConfig),
      },
    },
  },
}
```
Any suggestion?

Thanks.

**Comment by fredrikaverpil** [comment]
This indicates that the output is truncated (likely by your terminal backend). Are you on Windows/Powershell?

If not, would be helpful to know more about your OS/terminal. 

There's an open PR describing how to configure gotestsum, which solves this for Windows/Powershell. Maybe give it a try.

Here's a closed issue where a similar problem was observed: https://github.com/fredrikaverpil/neotest-golang/issues/147

**Reply by zivkovicn** [reply]
Hey, thanks for quick response and sry for not providing more details:
- macbook pro Apple M2 Pro
- osx sonoma 14.6.1
- wezterm (termnial)

checkhealth output:
```
neotest-golang: require("neotest-golang.health").check()

Requirements ~
- OK Binary 'go' found on PATH: /opt/homebrew/bin/go
- OK Found go.mod file for xxxx.go in xxxx/go.mod
- OK Treesitter parser for go is installed
- OK neotest is available
- OK nvim-treesitter is available
- OK nio is available
- OK plenary is available

DAP (optional) ~
- OK Binary 'dlv' found on PATH: /Users/nikola/.local/share/nvim/mason/bin/dlv
- OK dap is available
- OK dapui is available
- OK dap-go is available
```

feel free to ask for anything else if needed?  

edit: I just tried running this inside the alacritty terminal as well, still getting those warnings

**Reply by zivkovicn** [reply]
@fredrikaverpil I just tried with changing runner to be `gotestsum` and it is working as expected! Thank you for the hint, just would still like to figure out (if possible) why it is behaving like described with `go` as a runner.

**Reply by fredrikaverpil** [reply]
Yeah this is a little strange. I'm on macOS and wezterm myself and I've never seen these problems on my end.

Could you please try to enable debug logging (details in README) and supply the log here?
